orchestration/dagster_orchestration/hca_orchestration/resources/bigquery.py

```diff
@@ -0,0 +1,18 @@
+from google.cloud.bigquery import Dataset, Client
+from dagster import resource, InitResourceContext
+from hca_orchestration.contrib.google import get_credentials
+
+
+@resource
+def bigquery_client(init_context: InitResourceContext) -> Client:
+    return Client(credentials=get_credentials())
```

Comment:
do we anticipate needing to configure multiple sets of credentials at any point, and would it help to add support for that here?

Reply:
No. All of these are running as the same SA which we'll give the appropriate permissions.